src/main/res/values/strings.xml

```diff
@@ -296,6 +296,12 @@ Please report any problems you encounter via Github.</string>
 
     <string name="export_config">Export Configuration</string>
 
+    <string name="experimental_settings">Experimental</string>
+
+    <string name="keep_wakelock_while_binary_running">Keep Wakelock</string>
+
+    <string name="keep_wakelock_while_binary_running_summary">Keep the CPU awake while Syncthing is running. Use this setting if you experience unexpected disconnects while operating on battery. This will result in increased battery consumption.</string>
```

Comment:
I would just use "Keep the CPU awake while Syncthing is running" as the title. Means the description isn't so long, and most people don't know what a wakelock is anyway.